### ISSUE TITLE
Thoughts about Moving to Gitlab

### ISSUE THREAD

**Issue by kevin51jiang** [RipMeApp/ripme#661]
Just an thread for people who want to discuss the pros and cons of moving away from github.

This is not an endorsement of staying or leaving, it's just an open discussion space.

**Comment by cyian-1756** [comment]
I don't think we should move the repro unless github starts doing shady things, but I think setting up a mirror of ripme at gitlab would be a good idea

**Comment by BBloggsbott** [comment]
I agree with @cyian-1756. I don't see a reason to move away from GitHub. It satisfies the needs of this project.

**Comment by bored** [comment]
I think it's far too early to consider migrating to another service. Microsoft have explicitly said they will run GitHub independently and some of their previous acquisitions suggest they won't go back on that statement (Mojan, SwiftKey, etc.). That being said I like the idea of mirroring the repository on GitLab, just in case GitHub 365 comes out :wink:

**Comment by BBloggsbott** [comment]
I feel that the only solution will be to import this repository to Gitlab and push it back to Github
